src/app/qgsoptions.h

```diff
@@ -43,6 +43,20 @@ class APP_EXPORT QgsOptions : public QgsOptionsDialogBase, private Ui::QgsOption
     Q_OBJECT
   public:
 
+    /**
+     * Behavior to use when encountering a layer with an unknown CRS
```

Comment:
Semantically, is it an unknown CRS, or a layer with missing CRS?

Reply:
I guess it's either -- an un-readable CRS (e.g. a broken proj string, or WKT which can't be parsed) results in an invalid CRS, so the same situation as if the layer had no CRS to begin with.

Reply:
Makes sense.

Reply:
So do you think a wording change is required here?

Reply:
Na, I see your point now, it's fine.

IMHO, this is ready to merge, and as you pointed out, we'll need all the time between now and release day to stress test this most needed UI/UX behavior fix.

Reply:
Ok thanks... let's go!